core/components/atoms/card/__stories__/style.css

```diff
@@ -0,0 +1,7 @@
+.Cards {
+    height: 250px;
+}
+
+.Card-list {
+    height: 350px;
```

Comment:
Height should be more than 40vh for this example.

Reply:
350px is greater than 40vh should I convert px into vh?

Reply:
so I need to increase the height of the table? Should I use 100vh className?

Reply:
you can, but the example should fit properly and scroll should be there for large screens also.